pages/vi/profiles/rohansuri17.md

```diff
@@ -0,0 +1,25 @@
+# Rohan Suri #
+Denver,CO MST 
+Ubuntu 20.04 LTS
+
+![Profile Picture](https://github.com/rohansuri17/rohansuri17-OLE.github.io/blob/master/49506437_2602189069821948_6174767657032613888_n.jpg)
```

Comment:
@rohansuri17 think it is better to add this picture to the repo ...

Reply:
Looks good to me!